cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -119,7 +118,7 @@ public Boolean call() throws Exception {
           throw Throwables.propagate(ex);
```

Comment:
hmmm, ``fetchAccessToken`` throws ``IOException``, and the logic is to only throw ``ServiceUnavailableException`` out of this method. Maybe we can just catch ``IOException``?